Add glmmTMB to GLM intro slides (#75)
courtiol/LM2GLMM: vignettes/GLM_intro_course.Rmd

## Hurdle fit of zero-augmented data `r .emo("practice")`

> You can fit hurdle models with `glmmTMB::glmmTMB()`. glmmTMB is extremely flexible package that is still relatively young but that looks very promising.

Reviewer: ```suggestion
You can fit hurdle models with `glmmTMB::glmmTMB()`.  **{glmmTMB}* is extremely flexible package that is relatively similar to **{spaMM}** (more details later).
```

> You can fit hurdle models with `glmmTMB::glmmTMB()`.  **{glmmTMB}** is extremely flexible package that is relatively similar to **{spaMM}** (more details later).

Reviewer: ```suggestion
You can fit hurdle models with `glmmTMB::glmmTMB()`.  **{glmmTMB}** is an extremely flexible package that is relatively similar to **{spaMM}** (more details on differences between them later).
```

```{r hurdle, message = FALSE}

> library(glmmTMB)
> fit_Zhurd1 <- glmmTMB(eggs ~ humans_eaten, ziformula = ~humans_eaten,
>                       family = glmmTMB::truncated_poisson(link = "log"), data = AliensZ)
> fit_Zhurd2 <- glmmTMB(eggs ~ humans_eaten, ziformula = ~1,
>                       family = glmmTMB::truncated_poisson(link = "log"), data = AliensZ)
> lmtest::lrtest(fit_Zhurd1, fit_Zhurd2)

Reviewer: ```suggestion
library(glmmTMB)
fit_Zhurd1 <- glmmTMB(eggs ~ humans_eaten, ziformula = ~humans_eaten,
                      family = truncated_poisson(link = "log"), data = AliensZ)
fit_Zhurd2 <- glmmTMB(eggs ~ humans_eaten, ziformula = ~1,
                      family = truncated_poisson(link = "log"), data = AliensZ)
lmtest::lrtest(fit_Zhurd1, fit_Zhurd2)
```

Author: No issues, just thought it could be good to make it that it's not a 'standard' family from {stats}. Happy to remove though.

Let's also compare the model parameter estimates:
```{r comp zeroinfl2}
tab <- rbind(Poisson = c(fit_Zpoiss$coefficients, NA, NA),
             Hurdle = unlist(fixef(fit_Zhurd1)),  ## NOTE: the hurdle part predicts positive counts, not zeros!!

>              ZeroInfl = unlist(fixef(fit_Zzi1)),  ## NOTE: the binary part predicts zeros, not counts!!

Reviewer: One of the comment must be wrong... I guess it predicts zeros...
